.github/workflows/release.yml

```
@@ -10,9 +10,14 @@ env:
   TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
   TURBO_TEAM: svelte
 
+permissions: {}
 jobs:
   release:
     # prevents this action from running on forks
+    permissions:
+      contents: write # to create release (changesets/action)
+      pull-requests: write # to create pull request (changesets/action)
+
     if: github.repository == 'sveltejs/kit'
```

the comment belongs to the if: condition, lets keep it that way
```suggestion
    # prevents this action from running on forks
    if: github.repository == 'sveltejs/kit'
    permissions:
      contents: write # to create release (changesets/action)
      pull-requests: write # to create pull request (changesets/action)
```